debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java

```diff
@@ -768,6 +768,7 @@ else if (inconsistentSchemaHandlingMode == EventProcessingFailureHandlingMode.WA
         }
         else {
             logger.debug("Skipping {} event: {} for non-monitored table {}", typeToLog, event, tableId);
```

Comment:
Probably worth to change the log message into "Filtering out {} event: ...", in order to be consistent with the usage of "skip" vs. "filter" in the metrics names. Or does it sound too weird for native English speakers?